pyomo/contrib/pynumero/doc/source/index.rst

```diff
@@ -0,0 +1,32 @@
+PyNumero Documentation
+=========================
+
+.. toctree::
+   :maxdepth: 1
+
+   overview
+   installation
+   examples
+   API documentation <apidoc/pynumero>
+   license
+   developers
+
+Sandia Funding Statement
```

Comment:
@jsiirola I don't think we have this in the Pyomo Sphinx documentation, should it be included here? Should it be added to the Pyomo Sphinx documentation?